src/components/Watch.svelte

```diff
@@ -86,6 +92,14 @@
     });
   };
   $: setTimeout(() => changeSide($videoSide, $chatSplit), 0);
+
+  function orientationChangeCallback() {
+    $videoSide = ($autoVertical && window.innerHeight > window.innerWidth) ? VideoSide.TOP : $videoSideSetting;
+  }
+  window.addEventListener('resize', orientationChangeCallback);
+  videoSideSetting.subscribe(orientationChangeCallback);
```

Comment:
``videoSideSetting`` is also subscribed in ``components/UISettings.svelte`` to set the value of ``videoSide`` whenever it changes. I'm confused.